packages/app/src/components/SearchBar/index.js

```diff
@@ -66,21 +67,53 @@ const SearchBar = () => {
 
   return (
     <>
-      <form onSubmit={handleSubmit}>
-        <fieldset>
-          <input
-            type="text"
-            name="search"
-            placeholder="  Search"
-            data-id="search-input"
-            onChange={handleInputChange}
-            aria-label="search-input"
-          />
-          <div data-id="error-message">
-            {showErrorMessage ? <>Application Not Found</> : null}
-          </div>
-        </fieldset>
-      </form>
+      <section className="searchBarWrapper">
+        <div className="row">
+          <form
+            className="column column-40 column-offset-25"
+            onSubmit={handleSubmit}
+          >
+            <fieldset>
+              <div className="searchBarInput">
```

Comment:
I think I see why the background-color was behaving weirdly.

What you had pushed up previously (where there was an extra white margin below the input) was applying `background: white` to the `searchBarInput` class, which is defined as a `div` that wraps the input field.

If you gave the `input` a class name, you could target that directly and avoid applying a colored background to the margin.